### PR TITLE
New sun sensor test

### PR DESCRIPTION
**Pull request by thetazero**
Each sun sensor is attempted to be detected, and then we ensure that we have properly matched sun sensors.

We ensure matching by placing the -X board face down, and ensuring the -X sun sensor is reading the lowest lux value. 
If this is not the case we print which sun sensor read the lowest lux value.
An example of output is shown below:

image

### CODE REVIEW

File: state_machine/applications/system-check/main.py [hunk 1]
```diff
@@ -1,53 +1,19 @@
...
-    "Sun+Z_Dark": ("", None),
-    "Sun+Z_Light": ("", None),
-    "Sun+X_Dark": ("", None),
-    "Sun+X_Light": ("", None),
-    "CoilDriverX": ("", None),
-    "CoilDriverY": ("", None),
-    "CoilDriverZ": ("", None),
-    "Burnwire": ("", None),
-}
+result_dict = {}
```
Comment (jbwillis) [inline state_machine/applications/system-check/main.py:16]: Are there changes that need to be made to the other tests to handle this modification of `result_dict`?

**Review by jbwillis — CHANGES_REQUESTED** [review]
This looks fantastic. One question and then I'd like to run it on the hardware to verify it.

File: state_machine/applications/system-check/main.py (hunk 1 above)
Comment (thetazero) [reply state_machine/applications/system-check/main.py:16]: Nope

Comment (thetazero) [reply state_machine/applications/system-check/main.py:16]: It was used to print a list of tests that haven't been run, which we no longer do.

**Comment by thetazero** [comment]
> This looks fantastic. One question and then I'd like to run it on the hardware to verify it.

I've run it on the old cubesat to verify it, and it gives the correct results.

**Review by jbwillis — APPROVED** [review]

**Pull request merged**